src/app/app.component.html

```diff
@@ -1 +1,140 @@
+              <button class="navbar-toggler collapsed" type="button" aria-controls="navbar-default"
+                      aria-label="Toggle navigation"><span class="icon-bar top-bar mt-0"></span> <span
+                class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</header>
+<div class="border-bottom pb-lg-4 pb-3">
+  <nav class="navbar navbar-expand-lg navbar-light navbar-default pt-0 pb-0">
+    <div class="container px-0 px-md-3">
+      <div class="offcanvas offcanvas-start p-4 p-lg-0">
+        <div class="d-flex justify-content-between align-items-center mb-2 d-block d-lg-none"><a routerLink="/"><img
+          src="/assets/images/logo/freshcart-logo.svg" alt="eCommerce HTML Template"></a>
+          <button type="button" class="btn-close" aria-label="Close"></button>
+        </div>
+        <div class="d-block"></div>
+        <div class="row">
+          <ul class="navbar-nav">
+            <li *ngFor="let item of categories$ | async" class="nav-item"><a class="nav-link" routerLink="/categories/{{item.id}}"
+                                                                             role="button">{{item.name}}</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </nav>
+</div>
+
+<router-outlet></router-outlet>
+
+<footer class="footer fixed-bottom">
+
+  <div class="container">
+    <div class="row g-4">
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12"><h6 class="mb-4">Categories</h6>
+            <div *ngFor="let item of categories$ | async" class="nav-item"><a routerLink="/categories/{{item.id}}" class="nav-link">{{item.name}}</a></div>
+          </div>
+        </div>
+
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4 pb-4">
+          <div class="col-12"><h6 class="mb-4">Stores</h6>
+
+            <div *ngFor="let item of stores$ | async" class="nav-item"><a routerLink="/stores/{{item.id}}" class="nav-link">{{item.name}}</a></div>
+          </div>
+
+        </div>
+
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12"><h6 class="mb-4">Get to know us</h6>
+            <ul class="navbar-nav">
+              <li *ngFor="let item of FooterGetToKnowUs$ | async" class="nav-item"><a [routerLink]="''" class="nav-link">{{item}}</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+    </div>
+    <div class="border-top py-4">
+      <div class="row align-items-center">
+        <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+          <ul class="list-inline mb-0">
+            <li class="list-inline-item text-dark">Payment Partners</li>
+            <li class="list-inline-item"><a href="#!"><img src="../assets/images/payment/amazonpay.svg" alt=""></a></li>
```

Comment:
use absolute path in whole PR

Reply:
I'm sorry but I don' understand. Could you explain to me what do you mean?

Reply:
Look at the value in the src attribute. You have "../" and it is a relative path because you are pointing to a location from the current file. But absolute path is every time this same because absolute path is start from project folder.

Read about: Absolute path vs. relative path